### PR TITLE
Solve #14

### PR DESCRIPTION
**Pull request by allopez7**

### CODE REVIEW

File: test/14.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const solution = require('../solutions/14.js');
+
+const tester = (upTo, i=2)=>{
```
Comment (seemcat) [inline test/14.js]: why is your test so complicated? [Look at mine it's so simple](https://github.com/seemcat/algorithm/blob/0b35dadac4ba9060fc379304f366a1b28588718e/test/8.js). You don't need to recurse.

**Review by seemcat — CHANGES_REQUESTED** [review]

File: test/14.js (hunk 1 above)
Comment (allopez7) [reply test/14.js]: This is the way Royce taught me. Song told him to teach me so I could teach everyone else.

Comment (allopez7) [reply test/14.js]: Ok, I see the problem, no array! Will fix.

Comment (allopez7) [reply test/14.js]: Done!

**Review by roycep — APPROVED** [review]
Good job Alberto! Nice way to compare results!!

**Pull request merged**